app/src/main/java/movierating/services/ReviewServiceImpl1.java

```diff
@@ -19,7 +21,7 @@ public ReviewServiceImpl1(UserRepository userRepository, MovieRepository movieRe
 
 
     @Override
-    public void addReview(String userName, String movieName, int rating) {
+    public void addReview(String userName, String movieName, int rating) throws MultipleReviewsException, MovieNotReleasedException {
```

Comment:
When is MultipleReviewsException being thrown?

Reply:
Changed this for now.